docs/sources/operator/operator-integrations.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Set up Agent Operator integrations
+weight: 350
+---
+# Set up Agent Operator integrations
+
+This topic provides examples of setting up Agent Operator integrations, including [node_exporter](#set-up-an-agent-operator-node_exporter-integration) and [mysqld_exporter](#set-up-an-agent-operator-mysqld_exporter-integration).
+
+## Before you begin
+
+Before you begin, make sure that you have deployed the Grafana Agent Operator CRDs and installed Agent Operator into your cluster. See [Install Grafana Agent Operator with Helm]({{< relref "./helm-getting-started.md" >}}) or [Install Grafana Agent Operator]({{< relref "./getting-started.md" >}}) for instructions.
+
+Also, make sure that you [deploy the GrafanaAgent resource]({{< relref "./deploy-agent-operator-resources.md" >}}) and the `yaml` you use has the `integrations` definition under `spec`.
```

Comment:
@marctc , does this information belong here: "the `yaml` you use has the `integrations` definition under `spec`?"  Should we add this within the steps?

Reply:
I assume that users have followed previous steps (before you begin). This is just a reminder for that